ACI.txt

```diff
@@ -375,7 +375,7 @@ aci: (targetattr = "audio || businesscategory || carlicense || departmentnumber
 dn: dc=ipa,dc=example
```

Comment:
You also need to update VERSION.m4 since you're adding new commands.

Reply:
I always forget about ``VERSION.m4``. Fixed!